vllm_spyre/platform.py

```diff
@@ -200,6 +200,13 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
         # set env vars for torch_sendnn to consume
         os.environ["VLLM_DT_MAX_CONTEXT_LEN"] = str(
             vllm_config.model_config.max_model_len)
+        if (envs_spyre.VLLM_SPYRE_USE_CB
+                and vllm_config.model_config.max_model_len > 32 * 1024):
+            logger.warning(
+                'Max context length too big. Currently only 32K (32768) ' \
```

Comment:
Nit:  missing a verb, also for me long would sounds better than big.
```suggestion
                'Max context length is too big. Currently only 32K (32768) ' \
```